self_start/youtube.md

```diff
@@ -29,3 +29,11 @@ If you have come here, you probably already know what neural networks are. The f
 Now that you know how to implement a neural network from scratch (good job btw), it's time to get started with high level deeplearning libraries. But which one? There are plenty to choose from; Tensorflow, Keras, Pytorch, CNTK, MxNet.. Let's ask my main man [Siraj](https://www.youtube.com/channel/UCWN3xxRkmTPmbKwht9FuE5A):
 
 [![Keras Explained](http://img.youtube.com/vi/j_pJmXJwMLA/0.jpg)](http://www.youtube.com/watch?v=j_pJmXJwMLA "Keras Explained")
+
```

Comment:
Machine learning?
[A Friendly Introduction to Machine Learning](https://youtu.be/IpGxLWOIZy4)
[What is Artificial Intelligence (or Machine Learning)?](https://youtu.be/mJeNghZXtMo)
Quickstart series from Google developers
[Hello World - Machine Learning Recipes](https://youtu.be/cKxRvEZd3Mw)